src/metabase/models/dashboard.clj

```diff
@@ -436,6 +436,7 @@
   (-> (into (sorted-map) dashcard)
       (dissoc :id :collection_authority_level :dashboard_id :updated_at)
       (update :card_id                serdes.util/export-fk 'Card)
+      (update :action_id              serdes.util/export-fk 'Action)
```

Comment:
A singular Dashboard action? I don't know much about the design of Actions, but this surprised me.